app/forms/provider_interface/offer_wizard.rb

```diff
@@ -90,6 +98,10 @@ def no_ske_required?
       ActiveModel::Type::Boolean.new.cast(@ske_required).blank?
     end
 
+    def no_languages_ske_required?
+      ske_language_required.include?('no')
```

Comment:
Should we change this to `== ['no']`?  And/or validate that it doesn't contain `'no'` and something else as well?